SPARQLWrapper/main.py

```diff
@@ -73,6 +82,18 @@ def parse_args(test=None):
         help="sparql endpoint",
         default="http://dbpedia.org/sparql",
     )
+    parser.add_argument(
```

Comment:
This may be very minor, but it may be better to rather make the default None, and then only set it the method if it was actually specified, so that we can leave open the logic for picking the default method to sparqlwrapper.

Reply:
Similar to what you did for auth